controllers/comment-controllers.js

```diff
@@ -0,0 +1,43 @@
+const { deleteComment, addComment } = require("../models/comment-models");
+const {
+  checkCommentIdExists,
+  checkIdExists,
+  checkUserNameExist,
+} = require("../models/article-parameters-exists-models");
+
+const postComment = (request, response, next) => {
```

Comment:
Looks like you have pushed two tasks on one branch. Please try to avoid this

Reply:
Sorry! Merge issues